v2/internal/config/vars_internal_test.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/Azure/azure-service-operator/v2/pkg/common/config"
 )
 
-//nolint:paralleltest
```

Comment:
Did we lose the parallel test linter?

Reply:
No, it got upgraded to detect SetEnv so the suppression isn't needed any longer. See [PR#33](https://github.com/kunwardeep/paralleltest/pull/33)